DMX.Core.Api/Brokers/Storages/StorageBroker.Labs.cs

```diff
@@ -20,8 +20,13 @@ public partial class StorageBroker
         public IQueryable<Lab> SelectAllLabsWithDevices() =>
             SelectAll<Lab>().Include(lab => lab.Devices);
 
-        public async ValueTask<Lab> SelectLabByIdAsync(Guid labId) =>
-            await FindAsync<Lab>(labId);
+        public async ValueTask<Lab> SelectLabByIdAsync(Guid labId)
+        {
+            Lab lab = await SelectAsync<Lab>(labId);
+            this.Entry(lab).Collection(lab => lab.Devices);
```

Comment:
The acceptance test is failing because when we delete a lab then try to retrieve that same lab (expecting a 404), `lab` is null which causes exception to be thrown. But introducing a null check here violates adding business logic to the broker. Need to think on this...